jenkins-pipelines/features-scale-450-500-cluster-test.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    params: params,
+
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'grow_cluster_test.GrowClusterTest.test_grow_x_to_y',
+    test_config: '''["test-cases/features/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
+
+    timeout: [time: 27500, unit: 'MINUTES'],
```

Comment:
this is less then the test_duration in this case

Reply:
FIxed